.github/workflows/js_tests.yml

```diff
@@ -0,0 +1,272 @@
+name: JS tests master
+
+on:
+  push:
+    branches:
+      - master
+      - 3.x-maintenance
+    paths:
+      - '.github/scripts/*.sh'
+      - '.github/workflows/*.yml'
+      - 'gui/package.json'
```

Comment:
*.less

Reply:
*.ttf

Reply:
*.json

Reply:
> run gui/package.json in CI since it has been broken for ages.

Is that worth fixing?

Why are any of the other extensions relevant? we are not even running tests at this point on JS

Reply:
The json things are included by webpack, i.e. they change stuff. Especially if a .json is removed builds can break.

I didn't realise we didn't run the JS tests. But they exist and we probably should: https://github.com/lektor/lektor/blob/master/Makefile#L32

I think we should probably scrap `gui/` and rebuild it from scratch. It's based on an ancient version of Electron.

Reply:
The same as json goes for less files.

Reply:
Got it.